tc/sgx/trusted_worker_manager/enclave/work_order_data_handler.cpp

```diff
@@ -120,41 +118,46 @@ namespace tcf {
     }
 
     void WorkOrderDataHandler::ComputeHashString(std::string session_key_iv) {
-        ByteArray iv_byte;
+        ByteArray iv_bytes;
         if (iv.empty() && (enc_data_key_str.empty() || "null" == enc_data_key_str)) {
             iv = session_key_iv;
         }
-
-        std::string encrypted_output_str = EncryptOutputData(iv);
+        iv_bytes = HexStringToBinary(iv);
```

Comment:
As per spec, default value of iv is zero. This function should assign iv=0 if iv is empty

Reply:
ok